src/main/java/com/titarenko/dao/JdbcEmployeeDaoImpl.java

```diff
@@ -26,8 +29,13 @@ public Integer create(Employee employee) {
             preparedStatement.setString(3, employee.getPosition());
             preparedStatement.setInt(4, employee.getSalary());
             preparedStatement.setDate(5, Date.valueOf(employee.getDateOfHire()));
-            if (preparedStatement.executeUpdate() != 0) {
+
+            if (validator.isValidEmployee(employee)) {
```

Comment:
I'm okay with that right now, but:
It is not very good to add any logic to DAO layer. This class has to implement `EmployeeDao` interface and provide **only** CRUD operations without any business logic.
Let's assume, that you need to create one more implementation of `EmployeeDao` - in-memory DB, for example. And you will be forced to do the same validations as here.
Also, it is a bad practice to call dao methods directly.
That's why there is should be `EmployeeService` between upper service (`Menu` in our case) and DAO.